test/core/bulk.wast

```diff
@@ -128,14 +128,14 @@
 (assert_return (invoke "load8_u" (i32.const 1)) (i32.const 0xcc))
 (assert_return (invoke "load8_u" (i32.const 2)) (i32.const 0))
 
-;; Init ending at memory limit and segment limit is ok.
-(invoke "init" (i32.const 0xfffc) (i32.const 0) (i32.const 4))
-
-;; Out-of-bounds writes trap, but all previous writes succeed.
+;; Out-of-bounds writes trap, and nothing is written.
 (assert_trap (invoke "init" (i32.const 0xfffe) (i32.const 0) (i32.const 3))
     "out of bounds memory access")
-(assert_return (invoke "load8_u" (i32.const 0xfffe)) (i32.const 0xaa))
-(assert_return (invoke "load8_u" (i32.const 0xffff)) (i32.const 0xbb))
+(assert_return (invoke "load8_u" (i32.const 0xfffe)) (i32.const 0x00))
+(assert_return (invoke "load8_u" (i32.const 0xffff)) (i32.const 0x00))
+
+;; Init ending at memory limit and segment limit is ok.
```

Comment:
May be preferable to leave this where it was before and then tweak the two asserts above to check for 0xcc and 0xdd?